frontend/src/components/4.Preferences/Preferences.js

```diff
@@ -30,6 +30,9 @@ function Preferences() {
             fontSize={3}
           ></UserInput>
         </div>
+        <Link to="/Lobby">
+          <button className="GoButton">Go</button>
+        </Link>
 
         <button className='GoButton'>Go</button>
```

Comment:
Do we need this second `Go` button? Cause it shows up as two when you run it

Reply:
Oh weird I thought that was in the main, maybe I accidentally copy and pasted in the preference file.  Thanks, I have removed the duplicate!